Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM library/node:6.3
+
+WORKDIR /src
+
+COPY package.json .
+
+RUN npm install
+
+COPY . .
+
+RUN node_modules/bower/bin/bower --allow-root install \
```

Comment:
actually it should happen post npm-install thanks to the "postinstall" script in package.json